test/staking/StakingDistributor.test.ts

```diff
@@ -9,341 +9,395 @@ import {
-            await expect(
-                new Distributor__factory(owner).deploy(
-                    ZERO_ADDRESS,
-                    ohmFake.address,
-                    staking.address,
-                    authority.address
-                )
-            ).to.be.reverted;
-        });
-
-        it("does not accept 0x0 as OHM address", async () => {
-            await expect(
-                new Distributor__factory(owner).deploy(
-                    treasuryFake.address,
-                    ZERO_ADDRESS,
-                    staking.address,
-                    authority.address
-                )
-            ).to.be.reverted;
-        });
-
-        it("does not accept 0x0 as staking address", async () => {
-            await expect(
-                new Distributor__factory(owner).deploy(
-                    treasuryFake.address,
-                    ohmFake.address,
-                    ZERO_ADDRESS,
-                    authority.address
-                )
-            ).to.be.reverted;
-        });
     });
 
     describe("post-construction", () => {
-        beforeEach(async () => {
+        const bounty = 1000000
+
+        before(async () => {
             distributor = await new Distributor__factory(owner).deploy(
-                treasuryFake.address,
-                ohmFake.address,
+                treasury.address,
+                ohm.address,
                 staking.address,
-                authority.address
+                authority.address,
+                "2000"
             );
+
+            await treasury.connect(governor).enable("8", distributor.address, addressZero);
+
+            await staking.connect(governor).setDistributor(distributor.address);
         });
 
-        describe("distribute", () => {
-            it("will do nothing if there are no recipients", async () => {
-                await distributor.connect(staking).distribute();
+        beforeEach(async () => {
+            await distributor.connect(governor).setPools([]);
+        });
 
-                expect(treasuryFake.mint).to.have.callCount(0);
+        describe("triggerRebase", () => {
+            it("can be called by anyone", async () => {
+                await expect(distributor.connect(other).triggerRebase()).to.not.be.reverted;
+
+                await expect(distributor.connect(governor).triggerRebase()).to.not.be.reverted;
             });
 
-            it("can only be called by the staking contract", async () => {
+            it("mints to pools", async () => {
+                await advanceEpoch();
+                const pools = ["0x055475920a8c93cffb64d039a8205f7acc7722d3"];
+                await distributor.connect(governor).setPools(pools);
+
+                let [reserve0, reserve1, timestamp] = await ohmDai.getReserves();
+                const priceBefore = reserve1 / (reserve0 * 1000000000);
+                const balanceBefore = await ohm.balanceOf(pools[0]);
```

Comment:
can you add a expectedBalanceAfter = balanceBefore * rewardRate ?

everything else looks good, just a good sanity check here rather than vaguer >